src/EnergyPlus/HeatPumpWaterToWaterCOOLING.cc

```diff
@@ -906,7 +906,7 @@ void GshpPeCoolingSpecs::update(EnergyPlusData &state)
         // set node flow rates;  for these load based models
         // assume that the sufficient Source Side flow rate available
 
-        Real64 const ReportingConstant = state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
+        Real64 const ReportingConstant = state.dataHVACGlobal->TimeStepSys * Constant::SecInHour;
```

Comment:
Future replacement using state.dataHVACGlobal->TimeStepSysSec

Reply:
I could swear that I search-replaced for this exact idiom. Maybe it got messed up during a merge/conflict. There was at least one check-in I did that had a ton of conflicts.